src/main/java/io/specto/hoverfly/junit/api/HoverflyClient.java

```diff
@@ -94,62 +93,4 @@
      */
     boolean getHealth();
 
-    /**
-     * Static factory method for creating a {@link Builder}
-     * @return a builder for HoverflyClient
-     */
-    static Builder custom() {
-        return new Builder();
-    }
-
-    /**
-     * Static factory method for default Hoverfly client
-     * @return a default HoverflyClient
-     */
-    static HoverflyClient createDefault() {
-        return new Builder().build();
-    }
-
-    /**
-     * HTTP client builder for Hoverfly admin API
-     */
-    class Builder {
-
-        private String scheme = HoverflyConstants.HTTP;
-        private String host = HoverflyConstants.LOCALHOST;
-        private int port = HoverflyConstants.DEFAULT_ADMIN_PORT;
-        private String authToken = null;
-
-        Builder() {
-        }
-
-        public Builder scheme(String scheme) {
-            this.scheme = scheme;
-            return this;
-        }
-
-        public Builder host(String host) {
-            this.host = host;
-            return this;
-        }
-
-        public Builder port(int port) {
-            this.port = port;
-            return this;
-        }
-
-        /**
-         * Get token from environment variable "HOVERFLY_AUTH_TOKEN" to authenticate with admin API
-         * @return this Builder for further customizations
-         */
-        public Builder withAuthToken() {
-            this.authToken = System.getenv(HoverflyConstants.HOVERFLY_AUTH_TOKEN);
-            return this;
-        }
-
-        public HoverflyClient build() {
-            return new OkHttpHoverflyClient(scheme, host, port, authToken);
-        }
-    }
-
```

Comment:
We should keep it as it is a public API which user of hoverfly-java depends on. For example, people use hoverfly java may do the following to create a hoverfly client:
```
HoverflyClient.createDefault()
```

Why not create a new builder method to set config from a `HoverflyConfiguration ` object, something like this: 
```
HoverflyClient.custom().withConfig(configuration);
```